vignettes/policy-files.Rmd

```diff
@@ -18,7 +18,9 @@ There are two ingredients that are needed to calculate course grades with `grade
 
 Policy files are stored as [.yaml](https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started) files, a simple format for storing hierarchical data in a text file with minimal markup. Here is an example of a policy file for a course with a fairly simple grading scheme.
 
-```yaml
+Note that category names cannot be the same as any assignment names.
+
+``` yaml
```

Comment:
The code chunk will run if the language is surrounded in curly braces. E.g. {r}.